src/js/components/Ballot/OfficeItemCompressed.jsx

```diff
@@ -403,8 +403,8 @@ class OfficeItemCompressed extends Component {
           {/* *************************
             Display either a) the candidates the voter supports, or b) the first several candidates running for this office
             ************************* */}
-          <Container>
-            { this.state.candidateList.map((oneCandidate) => {
+          <Container candLength={candidateList.length}>
```

Comment:
Please use full-word variables. "candidateLength" instead of "candLength". I go into more depth regarding why here: https://github.com/wevote/WebApp/blob/develop/docs/contributing/CONTRIBUTING_STANDARDS.md